sematic/ui/src/components/Notes.tsx

```diff
@@ -50,7 +50,17 @@ export function NoteView(props: { note: Note; author: User }) {
           }}
         >
           <TimeAgo date={note.created_at} /> on run{" "}
-          <code style={{ fontSize: 12 }}>{note.root_id.substring(0, 6)}</code>
+          <code
+            style={{
+              fontSize: 12,
+              color:
+                rootRun.id == note.root_id
```

Comment:
What's the reason for the styling change here? You can still post notes from child runs, right?

Reply:
Yes, but we only display the root run ID on the note.